tdrs-backend/tdpservice/search_indexes/parsers/validators/category2.py

```diff
@@ -0,0 +1,49 @@
+"""Validate category 2 errors."""
+from tdpservice.search_indexes.parsers.validators.validator import FatalEditWarningsValidator
+
+
+def validate_cat2(name: str, value: str, condition: dict, model_obj) -> tuple:
+    """Validate categoy 2 errors."""
+    if name in condition.keys():
+        schema = condition
+    else:
+        schema = {name: condition}
+    document = {name: value}
+
+    errors = validate(schema, document)
+    return errors
+
+def validate(schema, document):
+    """Validate the a document."""
+    validator = FatalEditWarningsValidator(schema)
+    validator.allow_unknown = True
+    validator.validate(document)
+
+    return validator.errors
+
+def t1_006(model_obj):
+    """Validate model_obj.RPT_MONTH_YEAR for year."""
+    # TODO discuss how we should address this edge case:
+    # t1_006 and t1_007 are both validating parts of the same field,
+    # Either we can pass though the specific name, "YEAR value from RPT_MONTH_YEAR",
+    # or we can pass the actual field name, "RPT_MONTH_YEAR".
+    # We could add custom logic to the validator to handle this case,
+    # but that would be a lot of work for only two validators.
+    # The logic would be similar to the logic in create_cat3_error where a custom error message is made.
+
+    name = "YEAR value from RPT_MONTH_YEAR"
+    value = int(str(model_obj.RPT_MONTH_YEAR)[0:4])
+
+    schema = {name: {'gte': 1998}}
```

Comment:
```suggestion
    schema = {name: {'gte': 2018}}
```

I recall previous conversations with @ADPennington that we don't want to accept files more than 5 years old at this time like we did with the year drop-down on the data files page.